uint8 map_control
uint8 navigation_task